website/addons/evernote/utils.py

```diff
@@ -0,0 +1,136 @@
+
```

Comment:
Much of this file will likely be removed after refactor, I'll review anything that remains after.

Also, this file should either start with an `import`, by specifying the coding (i.e. `# -*- coding: utf-8 -*-
`), or with module-level comments/docstrings, as per PEP8